reference-architecture/osp-cli/ansible.cfg

```diff
@@ -1,19 +1,16 @@
 # config file for ansible -- http://ansible.com/
 # ==============================================
 [defaults]
+remote_user = cloud-user
 forks = 50
 host_key_checking = False
-remote_user = cloud-user
 gathering = smart
 retry_files_enabled = false
 nocows = true
-#lookup_plugins = ./playbooks/lookup_plugins
-#log_path = /tmp/ansible.log
 
```

Comment:
I'd say to add an ansible_inventory configuration to avoid using "-i inventory" all the time.